qcsubmit/workflow_components/filters.py

```diff
@@ -349,18 +349,20 @@ def provenance(self) -> Dict:
 
 class RotorFilter(BasicSettings, CustomWorkflowComponent):
     """
-    Filters molecules based on the maximum allowed number of rotatable bonds.
+    Filters molecules based on the minimum/maximum allowed number of rotatable bonds.
 
     Note:
         Rotatable bonds are non terminal torsions found using the `find_rotatable_bonds` method of the
         openforcefield.topology.Molecule class.
     """
 
     component_name = "RotorFilter"
-    component_description = (
-        "Filter the molecules based on the maximum number of allowed rotatable bonds."
+    component_description = "Filter the molecules based on the minimum/maximum number of allowed rotatable bonds."
+    component_fail_message = "The molecule has too many(low) rotatable bonds."
+    minimum_rotors: int = Field(
```

Comment:
This should now have the type `Optional[int]` to show that `None` can be supplied. I think we should also explain what happens if `None` is supplied in the field description.